src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -30,32 +30,53 @@ const getSwitchAuthenticatorLink = (appState) => {
   return [];
 };
 
-const getForgotPasswordLink = (appState) => {
+const getForgotPasswordLink = (appState, settings) => {
   const forgotPasswordLink = {
     'type': 'link',
     'label': loc('forgotpassword', 'login'),
     'name': 'forgot-password',
   };
 
+  const customForgotPasswordHref = settings.get('helpLinks.forgotPassword');
+
   // at identify page when only Org Authenticator Password may be available
   if (appState.getActionByPath(ORG_PASSWORD_RECOVERY_LINK)) {
```

Comment:
I think it can be
```
if custom forgot password link
  return {}
else if (ORG_PASSWORD_RECOVERY_LINK) 
  return ..
else if (ENROLLED_PASSWORD_RECOVERY_LINK)
  return ..
```